en/docs-autocomplete.js

```diff
@@ -71,11 +92,11 @@ function setupAutocomplete(site){
 jQuery(function(){
 	
 
-	jQuery.getJSON(window._pageRelpath+'site.json',function(data){
+	jQuery.getJSON('/site.json',function(data){
```

Comment:
this isn't going to work, i think the relative path is necessary. /site.json will resolve to the site base url